x-pack/plugins/infra/public/pages/metrics/inventory_view/hooks/use_waffle_time.ts

```diff
@@ -42,7 +42,7 @@ export const useWaffleTime = () => {
   const { currentTime, isAutoReloading } = urlState;
 
   const startAutoReload = useCallback(() => {
-    setState((previous) => ({ ...previous, isAutoReloading: true }));
+    setState((previous) => ({ ...previous, isAutoReloading: true, currentTime: Date.now() }));
```

Comment:
It doesn't make sense to enable auto-refresh when `currentTime` is set to a date in the past.